packages/tokens/build.js

```diff
@@ -217,7 +223,12 @@ StyleDictionary.registerFormat({
       {},
     )
 
-    return JSON.stringify(sortTokensByKey(tokens), undefined, 2) + `\n`
+    const sortedTokens =
+      dictionary.allTokens?.[0].attributes?.category == 'spacing'
+        ? tokens
+        : sortTokensByKey(tokens)
+
+    return JSON.stringify(sortedTokens, undefined, 2) + `\n`
```

Comment:
Instead of this, might be clearer/cleaner to:
```suggestion
    // Leave spacing tokens sorted by size
    if (dictionary.allTokens?.[0].attributes?.category === 'spacing') {
      return JSON.stringify(tokens, undefined, 2) + `\n`
    }

    return JSON.stringify(sortTokensByKey(tokens), undefined, 2) + `\n`
```

Might also be more extensible in the future as other tokens like font size are likely to be similar.